sqldelight-gradle-plugin/src/test/fixtures/joining-column-wrong-name/failure.txt

```diff
@@ -0,0 +1,6 @@
+Table.sq line 8:59 - Table joined against does not contain a column with the name other_column
```

Comment:
Ah yeah, or subquery presumably.

On Fri, Mar 4, 2016 at 4:06 PM Alec Strong <notifications@github.com> wrote:

> In
> sqldelight-gradle-plugin/src/test/fixtures/joining-column-wrong-name/failure.txt
> <https://github.com/square/sqldelight/pull/172#discussion_r55088664>:
>
> > @@ -0,0 +1,6 @@
> > +Table.sq line 8:59 - Table joined against does not contain a column with the name other_column
>
> Unfortunately no since the table your joining against could itself be a
> join
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/sqldelight/pull/172/files#r55088664>.
>